Lg/0.6.4 release notes (#191)
dfinity/docs: modules/release-notes/pages/sdk-release-notes.adoc

[[highlights]]
== Highlights of what's new in {release}

The {release} release primarily includes internal improvements that are not user-facing.
The only user-facing features and fixes are the following:

- A new Reserved type has been added to the JavaScript agent library.

> - The Candid UI has been updated with a timer to await the results when issuing function calls.

Reviewer: ```suggestion
- Fixed the timer in Candid UI when issuing function calls.
```
